andela/AndelaEatsUI
# Fix bug in githook

#### What does this PR do?
Fix missing git hooks when running `git commit` command

#### Description of Task to be completed?
- move 'hooks' directory into the 'client' directory

#### How should this be manually tested?
- clone the project
- make changes then run `git commit -m "<your message>"` to see the git pre-commit hook run

#### What are the relevant pivotal tracker stories?
[#159135244](https://www.pivotaltracker.com/n/projects/2178749/stories/159135244)

#### Any background context you want to add ?
N/A

#### Screenshots
NA
